DataFormats/METReco/src/MET.cc

```diff
@@ -68,6 +68,24 @@ MET::MET( double sumet_, const std::vector<CorrMETData>& corr_,
     }
 }
 
+// Copy constructor
+//____________________________________________________________________________||
+MET::MET( const MET& met ): 
```

Comment:
Hi Slava,

the MET significance matrix was not set in case the slimmedMETs collection was recreated using miniAOD input. I think you are right and the important change is in DataFormats/PatCandidates/src/MET.cc , while the copy constructor in DataFormats/METReco/src/MET.cc is equivalent to the default copy constructor generated by the compiler. Let me know if you prefer that I remove the copy constructor from DataFormats/METReco/src/MET.cc . 

Cheers,


Christian